artistools/spectra/plotspectra.py

```diff
@@ -1341,6 +1341,10 @@ def addargs(parser) -> None:
         "--makevspecpol", action="store_true", help="Make file summing the virtual packet spectra from all ranks"
     )
 
+    parser.add_argument(
+        "--makevspecgrid", action="store_true", help="Make file summing the virtual packet grid from all ranks"
```

Comment:
Consider renaming vspecgrid to vspecvelgrid to be more descriptive. I plan to do this in the artis code at some point. Actually, I didn't know that anyone was using the vspecgrid functionality. It might be worth adding some extra em_pos, em_time columns to the vpackets*.out files so that we can do similar plots directly from that data.